docs/AccessTokenServiceApi.md

```diff
@@ -0,0 +1,32 @@
+# \AccessTokenServiceApi
```

Comment:
extra \\?

Reply:
All of the other `...ServiceApi.md` docs have a leading \, so I kept it in, should they be removed?